Makefile

```diff
@@ -48,23 +36,23 @@ endif
 # Environment
 BASE_PATH := $(shell pwd)
 HUGO_PATH := $(BASE_PATH)/.hugo
-HUGO_URL = github.com/spf13/hugo
 HUGO_NAME := hugo
-HUGO_URL_NAME := hugo_$(HUGO_VERSION)_$(URL_ARCH)-$(URL_OS)
 WORKDIR := $(PWD)
 BUILD_PATH := $(WORKDIR)/build
 LANDING_ARTIFACT := landing_$(TAG).tar.gz
 
 # Tools
-CURL = curl -L
-HUGO = $(HUGO_PATH)/$(HUGO_NAME)
-MKDIR = mkdir -p
-GIT = git
-YARN = yarn
-CGO_ENABLED=0
-REMOVE = rm -rf
-COMPRESS = tar -cf
-TAR = tar -zcvf
+CURL := curl -L
```

Comment:
`Simply expanded variables` use to be preferred over `recursively expanded` because it behavior is easily predictable, so they're not error prune and they also offer more performance [1](https://www.gnu.org/software/make/manual/html_node/Flavors.html) [2](http://pages.cs.wisc.edu/~alexande/books/gnu_make/make_6.html)